source/ftrack_connect_pipeline_qt/ui/assembler/assembler.py

```diff
@@ -181,15 +182,17 @@ def _on_dependencies_resolved_async(self, result):
                 self.stopBusyIndicator.emit()
         except RuntimeError as re:
             if str(re).find('Internal C++ object') == -1:
+                self.loadError.emit('An internal error occurred')
```

Comment:
A part from sending this signal, We should be raising an error here or at least logger it. Same for the others

Reply:
The error/exception is re-raised the line after